litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-DP-ON-SPECIFIC-NODE/test.yml

```diff
@@ -276,6 +276,28 @@
           register: node3
           failed_when: "node3.json.nodeType['mayadata.io/control-plane'] != 'true'"
 
+        ## Checking the openebs Installation    
+        - name: Fetch OpenEBS control plane pods state
+          shell: kubectl get pods -n {{ namespace }}  | grep {{ item }} | awk '{print $3}' | awk -F':' '{print $1}' | tail -n 1
+          register: app_status
+          until: app_status.stdout == 'Running'
+          with_items:
+            - "{{ openebs_components }}"
+          retries: 30
+          delay: 10
```

Comment:
@amitbhatt818 can we increase the retries and decrease the delays? 10 sec delay is a bit too long.

Reply:
Ok @harshshekhar15